styles/settings-view.less

```diff
@@ -698,6 +698,14 @@
   }
 }
 
+.package-updates-status-view
```

Comment:
Whoops, no idea how I missed that `{`.

You're right about the underline, I'll updated that.